tests/pipeline/test_template.py

```diff
@@ -0,0 +1,92 @@
+from pathlib import Path
+from unittest.mock import ANY, MagicMock
+
+import pytest
+from pytest_mock import MockerFixture
+from typer.testing import CliRunner
+
+from kpops.cli.main import app
+from kpops.component_handlers.helm_wrapper.helm import Helm
+
+runner = CliRunner()
+
+RESOURCE_PATH = Path(__file__).parent / "resources"
+PIPELINE_BASE_DIR = str(RESOURCE_PATH.parent)
+
+
+class TestTemplate:
+    @pytest.fixture
+    def run_command(self, mocker: MockerFixture) -> MagicMock:
+        return mocker.patch.object(Helm, "_Helm__execute")
+
+    def test_default_template_config(self, run_command: MagicMock):
+        result = runner.invoke(
+            app,
+            [
+                "generate",
+                "--pipeline-base-dir",
+                PIPELINE_BASE_DIR,
+                str(RESOURCE_PATH / "custom-config/pipeline.yaml"),
+                "--defaults",
+                str(RESOURCE_PATH / "no-topics-defaults"),
+                "--template",
+            ],
+            catch_exceptions=False,
+        )
+
+        run_command.assert_called_with(
+            [
+                "helm",
+                "template",
+                "resources-custom-config-app2",
+                "bakdata-streams-bootstrap/streams-app",
+                "--values",
+                ANY,
+                "--version",
+                ANY,
```

Comment:
is `ANY` needed for the version as well?

Reply:
check duplicate conversation with @raminqaf